js/ui/grid_core/ui.grid_core.keyboard_navigation.js

```diff
@@ -101,6 +101,11 @@ function isFixedColumnIndexOffsetRequired(that, column) {
     return result;
 }
 
+function shouldPreventScroll(that) {
```

Comment:
IMO - if function uses context, it should be a method.

Reply:
This is a private function that is not used outside the module. I think it is better to keep it as is.